tests/consensus/v2_test.go

```diff
@@ -24,3 +26,243 @@ func TestCountdownTimeoutToSendTimeoutMessage(t *testing.T) {
 	// This shows we are able to decode the timeout message, which is what this test is all about
 	assert.Regexp(t, "^Masternodes does not contain signer addres.*", err.Error())
 }
+
+// Timeout handler
+func TestTimeoutMessageHandlerSuccessfullyGenerateTCandSyncInfo(t *testing.T) {
+	blockchain, _, _, _ := PrepareXDCTestBlockChain(t, 11, params.TestXDPoSMockChainConfigWithV2Engine)
+	engineV2 := blockchain.Engine().(*XDPoS.XDPoS).EngineV2
+
+	// Set round to 1
+	engineV2.SetNewRoundFaker(utils.Round(1), false)
+	// Create two timeout message which will not reach timeout pool threshold
```

Comment:
Doesn't need this cmd

Reply:
why not? First two will not trigger anything, but the third will